AK/StringUtils.cpp

```diff
@@ -36,6 +36,16 @@ namespace AK {
 
 namespace StringUtils {
 
+static constexpr bool is_whitespace_character(char ch)
+{
+    return ch == '\t'
+        || ch == '\n'
+        || ch == '\v'
+        || ch == '\f'
+        || ch == '\r'
+        || ch == ' ';
```

Comment:
`#include <ctype.h>` ? this looks like `isspace`

Reply:
Not sure if I _can_ do that in that header, maybe there's a reason why it was like that?
I'll try

Reply:
Build seems okay with it, I'll just use isspace :shrug: